charts/kueue/values.yaml

```diff
@@ -1,64 +1,95 @@
 # Default values for kueue.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+
+# -- Override the resource name
 nameOverride: ""
+# -- Override the resource name
 fullnameOverride: ""
-# Enable each function, like kustomize https://github.com/kubernetes-sigs/kueue/blob/main/config/default/kustomization.yaml
+# -- Enable Prometheus
 enablePrometheus: false
-# Enable x509 automated certificate management using cert-manager (cert-manager.io)
+# -- Enable x509 automated certificate management using cert-manager (cert-manager.io)
 enableCertManager: false
-# Enable API Priority and Fairness configuration for the visibility API
+# -- Enable API Priority and Fairness configuration for the visibility API
 enableVisibilityAPF: false
-# Customize controllerManager
+# -- Enable KueueViz dashboard
+enableKueueViz: false
+# -- Kubernetes cluster's domain
+kubernetesClusterDomain: cluster.local
 controllerManager:
-  #featureGates:
+  # -- ControllerManager's feature gates
+  featureGates: []
   #  - name: PartialAdmission
   #    enabled: true
   manager:
-    # priorityClassName: "system-cluster-critical"
+    # -- ControllerManager's pod priorityClassName
+    priorityClassName:
```

Comment:
Hm, would like to clarify that. If using "" semantically the same as leaving it blank?

Asking cause I see we set `fullnameOverride: ""`, so I think if they are semantically equivalent, then it would be nice to align them.

Reply:
Leaving it blank results in a null value.

Reply:
ok, so leaving black is semantically different than "", and it is intentional that for `priorityClassName` we leave blank so that it is same as on 0.12, right?

Reply:
> Currently, there’s no difference between an empty line and null – both work the same.

What do you mean precisely by "both work the same"? Does it mean that the resulting instance of kueue-controller-manager looks the same - without priorityClassName set, or is it set as `priorityClassName`, but result in the same integer priority?
Did you check that on a live cluster?

Reply:
I think leaving `priorityClassName` blank is actually more intuitive, because it is less obvious why and at which layer the empty string is stripped and not set.